### DISCUSSION TITLE
Cannot double-click ord binary

### DISCUSSION THREAD

**Discussion by tbreki** [ordinals/ord#2412] [Q&A] [answered]
have set up bitcoin core and sparrow, downloaded ord and trying to follow instructions here https://docs.ordinals.com/guides/inscriptions.html

but i cant run any bitcoind commands
and i cant run any ord commands.. 

when i open ord app it does not let me write anything inside also..

can someone please help me fix this, been spending 2 days trying to fix this
<img width="593" alt="Screenshot 2023-02-11 at 08 22 35" src="https://user-images.githubusercontent.com/93223461/218248288-d27ae423-95aa-4365-aa0a-46fcc37038b9.png">

**Comment by kaspar-nl** [comment]
What is the complete command you are using to run `ord`?

**Comment by tbreki** [comment]
https://docs.ordinals.com/guides/inscriptions.html im using this guide was not able to call bitcoind -txindex . and any ord commands other than the download one... but i finally managed to run the bitcoind -txindex w the help of chatgtp, the blockchain is now syncing lets see if i can use the ord commands when its done :)

**Comment by camelKace** [comment]
Do you mind shedding light as to what helped you to run the command with chatgpt

**Comment by github-Dheeraj** [comment]
what was the search query from it @tbreki on chatGPT,

facing same issue

**Comment by camelKace** [comment]
@github-Dheeraj any luck?

**Comment by casey** [comment, marked as answer]
The `ord` binary must be run from within the terminal.
